src/main/java/com/spotinst/sdkjava/model/bl/azure/statefulNode/StatefulNodeTasksConfiguration.java

```diff
@@ -0,0 +1,105 @@
+package com.spotinst.sdkjava.model.bl.azure.statefulNode;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.enums.AzureStatefulNodeTaskTypeEnum;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class StatefulNodeTasksConfiguration {
```

Comment:
Missing fields as mentioned in the API class.

Reply:
Same as signals.enabledAt . These are internal fields.